src/faultlog/deconfig_records.cpp

```diff
@@ -113,12 +122,6 @@ void DeconfigRecords::populate(nlohmann::json& jsonNag)
                 deconfigJson["LOCATION_CODE"] = attrLocCode;
             }
 
-            std::stringstream ss;
-            ss << "0x" << std::hex << hwasState.deconfiguredByEid;
-            deconfigJson["PLID"] = ss.str();
-            deconfigJson["REASON_DESCRIPTION"] = getDeconfigReason(
-                static_cast<DeconfiguredByReason>(hwasState.deconfiguredByEid));
-
             json header = json::object();
             header["DECONFIGURED"] = std::move(deconfigJson);
```

Comment:
we will end up adding an empty json if no attributes are set? and we will get the list only if HWAS state is set right?

Reply:
yes, do you want some default value here?

Reply:
as discussed the deconfig record will be ignored if physical path is not found.